autogen/main.tf.tmpl

```diff
@@ -236,8 +240,27 @@ resource "google_compute_backend_service" "default" {
   }
   {% endif %}
 }
+{% else %}
+resource "google_compute_backend_bucket" "default" {
```

Comment:
We should do this as a for_each so multiple buckets can be provided.

Reply:
I agree, but if a user provides more than 1 bucket, they will then need to provide their own url map. Not sure how best to enforce this or communicate it.